tests/vision/checks/performance/single_dataset_scalar_performance_test.py

```diff
@@ -0,0 +1,92 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+
+import warnings
+
+import torch
+from hamcrest import (assert_that, calling, equal_to, greater_than_or_equal_to,
+                      raises)
+from ignite.metrics import Accuracy, Precision
+
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.vision.checks.performance.single_dataset_scalar_performance import \
+    SingleDatasetScalarPerformance
+
+
+def test_detection_deafults(coco_train_visiondata, mock_trained_yolov5_object_detection, device):
+    # Arrange
+    check = SingleDatasetScalarPerformance()
+
+    # Act
+    result = check.run(coco_train_visiondata, mock_trained_yolov5_object_detection, device=device)
+
+    # Assert
+    # scalar
+    assert_that(type(result.value['score']), equal_to(float))
+    # metric
+    assert_that(result.value['score'], greater_than_or_equal_to(0))
+
+
+def test_detection_w_params(coco_train_visiondata, mock_trained_yolov5_object_detection, device):
+    # params that should run normally
+    check = SingleDatasetScalarPerformance(reduce=torch.max, reduce_name='torch_max')
+    result = check.run(coco_train_visiondata, mock_trained_yolov5_object_detection, device=device)
+    assert_that(type(result.value['score']), equal_to(float))
```

Comment:
assert the value too and use close_to with precision of 0.001, it helps to see if anything is random in our code
(sry for duplicated comments)